igvm/hypervisor.py

```diff
@@ -817,3 +832,57 @@ def device_to_netcat(self, device, size, listener):
             '| /bin/nc.openbsd -q 1 {2} {3}'
             .format(device, size, *listener)
         )
+
+    def hv_pcpus_used_by_vm(self, vm) -> float:
+        """pCPUs used by VM
+
+        Calculate the physical CPUs the VM will use on a potential destination
+        hypervisor.
+
+        :param: vm: VM object
+
+        :return: Amount of pCPUs as float
+        """
+
+        vm_performance_value = vm.vm_performance_value()
+
+        hv_perffactor_dest = self.dataset_obj['cpu_perffactor'] / 1000
```

Comment:
Better comment here that we can right now not save floats in graphite cache of Serveradmin an therefore divide the value by thousand because we multiplied it in advance.

Reply:
Comment added